static/app/components/searchQueryBuilder/tokens/filter/valueListBox.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from '@emotion/styled';
+import {isMac} from '@react-aria/utils';
+
+import {ListBox} from 'sentry/components/compactSelect/listBox';
+import type {SelectOptionOrSectionWithKey} from 'sentry/components/compactSelect/types';
+import {Overlay} from 'sentry/components/overlay';
+import type {CustomComboboxMenuProps} from 'sentry/components/searchQueryBuilder/tokens/combobox';
+import {itemIsSection} from 'sentry/components/searchQueryBuilder/tokens/utils';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+interface ValueListBoxProps<T> extends CustomComboboxMenuProps<T> {
+  isMultiSelect: boolean;
+  items: T[];
+}
+
+export function ValueListBox<T extends SelectOptionOrSectionWithKey<string>>({
```

Comment:
I've been using the `customMenu` prop for other customized menus, so I did the same thing here for the value menu. It's mostly the same as the default combobox behavior but I didn't want to add more bloat to that one